src/Service.php

```diff
@@ -561,16 +572,16 @@ public function doDeleteFromIndex(Concrete $object)
      *
      * @param Concrete $object
      *
-     * @throws Exception
+     * @throws DoctrineDbalException
      */
     public function fillupUpdateQueue(Concrete $object)
     {
-        $db = \Pimcore\Db::get();
+        $db = Db::get();
         $idField = DataObjectService::getVersionDependentDatabaseColumnName('id');
         //need check, if there are sub objects because update on empty result set is too slow
-        $objects = $db->fetchFirstColumn('SELECT `'. $idField .'` FROM objects WHERE path LIKE ?', [$object->getFullPath() . '/%']);
+        $objects = $db->fetchFirstColumn('SELECT `'. $idField .'` FROM objects WHERE `path` LIKE ?', [$object->getFullPath() . '/%']);
```

Comment:
```suggestion
        $objects = $db->fetchFirstColumn('SELECT `id` FROM objects WHERE `path` LIKE ?', [$object->getFullPath() . '/%']);
```
$idField should be replace right?

Reply:
not here as we read data from `objects` table which can vary based on Pimcore version used.